stl/inc/bit

```diff
@@ -106,7 +116,17 @@ _NODISCARD constexpr _Ty rotl(const _Ty _Val, const int _Rotation) noexcept {
 template <class _Ty, enable_if_t<_Is_standard_unsigned_integer<_Ty>, int> _Enabled>

```

Comment:
This enable if is using a nonstandard form, this is preexisting though

Reply:
This is written this way in #310. And it is correct: there's a forward declaration with `= 0`, and the default can't be repeated.